static/index.css

```diff
@@ -0,0 +1,254 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/** Layout **/
+
+.layout {
+  padding-top: 110px;
+}
+
+.layout__header {
+  min-height: 110px;
+  top: 0px;
+  position: fixed;
+}
+
+.header {
+  width: 100%;
+  background: #FFF;
+}
+
+.content {
+  width: 100%;
+  height: 800px;
+  background: #FFF;
+}
+
+.sidebar {
+  height: 140px;
+  width: 100%;
+  background: #FFF;
+}
+
+.footer {
+  display: none;
+}
+
+@media (min-width: 768px) {
+  .layout {
+    padding-top: 0px;
+    display: grid;
+    grid-template-columns: auto 280px;
+    grid-template-rows: 80px auto 80px;
+    grid-template-areas:
+      "header sidebar"
+      "content sidebar"
+      "footer sidebar";
+  }
+
+  .header {
+    position: relative;
+    grid-area: header;
+  }
+
+  .content {
+    grid-area: content;
+    margin-top: 25px;
+  }
+
+  .sidebar {
+    grid-area: sidebar;
+    height: auto;
+  }
+
+  .footer {
+    grid-area: footer;
+    display: block;
+  }
+}
+
+/** Atoms **/
+.button {
+  background-color: lightblue;
+  display: inline;
+  width: 100%;
+  font-size: 16px;
+}
+
+.top-bar__navigation-icon {
+  border: none;
+  background-color: transparent;
+  font-size: 18px;
+}
```

Comment:
This is part of `top-bar` molecule so it should not be treated it as an atom and it **should be positioned near the rest of the code related with it**